src/components/DotIndicatorMessage.js

```diff
@@ -25,11 +25,16 @@ const propTypes = {
     // Additional styles to apply to the container */
     // eslint-disable-next-line react/forbid-prop-types
     style: PropTypes.arrayOf(PropTypes.object),
+
+    // Additional styles to apply to the text
+    // eslint-disable-next-line react/forbid-prop-types
+    textStyle: PropTypes.arrayOf(PropTypes.object),
```

Comment:
could we rename this `textStyles` as it's an array?

Reply:
updated